js/app.js

```diff
@@ -30,9 +30,29 @@ const products = [bag, banana, bathroom, boots, breakfast, bubblegum, chair, cth
 function appendRandomProduct () {
     const tempArray = [];
     const select = document.getElementById('select');
+
+    do {
+        const randomProduct = products[Math.floor(Math.random() * products.length)];
+        console.log('randomProduct:', randomProduct);
+        console.log('temparry:', tempArray);
+        if (!tempArray.includes(randomProduct)) {
+            const randomProductEle = randomProduct.render();
+            select.appendChild(randomProductEle);
+            randomProduct.wasDisplayed();
+            tempArray.push(randomProduct);
+        }
+    } while (tempArray.length < 3);
```

Comment:
Use a while loop to populate our tempArray instead of a for loop; because we don't know how many times we want to loop!